app/scripts/controllers/citationInfo.ctrl.js

```diff
@@ -7,6 +7,38 @@ angular.module('yourStlCourts').controller('CitationInfoCtrl', function (faqData
   ctrl.paymentUrl = "";
   ctrl.citationCourtLocations = {};
 
+  ctrl.groupCitationsByDL = function(){
+    var dlNum = "";
+    var dlState = "";
+    var groupedCitations = new Array();
+
+    for(var citationIndex = 0; citationIndex < citations.length; citationIndex++){
+      var groupedCitation = citations[citationIndex];
```

Comment:
In general, for loops aren't really used anymore unless the index is needed. I see in this case you're not using the index. Instead, just use `citations.forEach(function(citation) { ... });`